runtimes/common/src/fees.rs

```diff
@@ -202,6 +202,7 @@ mod tests {
 		type RuntimeOrigin = RuntimeOrigin;
 		type Block = Block;
 		type Nonce = u64;
+		type RuntimeTask = RuntimeTask;
```

Comment:
`()`

Reply:
Update in: [685b377](https://github.com/KILTprotocol/kilt-node/pull/655/commits/685b3771f014fc93dd39c31d8e5f1a670756e041)